### PR TITLE
simply xml config

### PR DESCRIPTION
**Pull request by maggarwal13**

### CODE REVIEW

File: whois-db/src/main/resources/applicationContext-whois-base.xml [hunk 1]
```diff
@@ -15,8 +15,7 @@
 
     <import resource="applicationContext-nrtm.xml"/>
 
-    <import resource="applicationContext-nrtm4.xml"/>
```
Comment (eshryane) [inline whois-db/src/main/resources/applicationContext-whois-base.xml:18]: Is it possible to isolate the NRTMv4 changes into a separate PR ?

Comment (maggarwal13) [reply whois-db/src/main/resources/applicationContext-whois-base.xml:18]: These are all inter connected and actually easy to move.  The ones which requires cnsiderable work and some brains I have decided to put them in different PR

Comment (maggarwal13) [reply whois-db/src/main/resources/applicationContext-whois-base.xml:18]: separated https://github.com/RIPE-NCC/whois/pull/1504

**Comment by maggarwal13** [comment]
IT tests passed

**Review by MiguelAHM — APPROVED** [review]

**Pull request merged**